pubsub/pubsub.go

```diff
@@ -166,6 +166,27 @@ func (t *Topic) As(i interface{}) bool {
 	return t.driver.As(i)
 }
 
+// ErrorAs converts err to provider-specific types.
+// See Topic.As for more details.
+// ErrorAs panics if target is nil or not a pointer.
```

Comment:
Be consistent with `blob`... I don't think panics in those scenarios does it?

Reply:
Could I change blob? Both should be consistent with `xerrors.As`. See https://godoc.org/golang.org/x/exp/xerrors#As. Note that there is now an [official proposal](https://github.com/golang/go/issues/29934) for this.

Reply:
Sure. And `runtimevar`.